app/jobs/document_proofing_job.rb

```diff
@@ -12,8 +12,7 @@ def perform(
     trace_id:,
     image_metadata:,
     analytics_data:,
-    flow_path:,
-    liveness_checking_enabled: nil
```

Comment:
Since this is a background we will need to leave this argument here. Otherwise when we are in a 50/50 state jobs spawned on old instances still passing this arg will throw an argument error if they get picked up by a worker running new code.

Reply:
Put back.